panda.config.ts

```diff
@@ -18,24 +11,14 @@ export default defineConfig({
 
   prefix: 'leather',
 
-  presets: [],
+  presets: ['@pandacss/dev', '@leather-wallet/panda-preset'],
```

Comment:
How come the pandacss/dev is needed here?

Reply:
It is indeed un-necessary, thanks. They recommend adding it if you want to keep their base styles but we don't need those. 

I had removed it when testing but forgot to take it out of here. Updated